addons/html_editor/static/src/main/link/link_popover.xml

```diff
@@ -40,7 +46,7 @@
                     </div>
                 </div>
 
-                <div class="col o_link_dialog_preview" style="max-width: 125px;border-left: 1px solid #DEE2E6;">
+                <div t-if="!state.isImage" class="col o_link_dialog_preview" style="max-width: 125px;border-left: 1px solid #DEE2E6;">
```

Comment:
why not move every think in the t-else ? (maybe swich <div t-else=""> => <t t-else=""><div>